src/vivarium_public_health/metrics/disease.py

```diff
@@ -134,3 +133,53 @@ def on_time_step_prepare(self, event: Event) -> None:
 
     def aggregate_state_person_time(self, x: pd.DataFrame) -> float:
         return len(x) * to_years(self.step_size())
+
+    ##################
+    # Report methods #
+    ##################
+
+    def report_person_time(self, measure: str, results: pd.DataFrame):
```

Comment:
It seems like there's a lot of duplicated code between these two report methods and also the report method of the `DisabilityObserver`

Reply:
I'll look and see if I can extract something out into a utility function